tests/unit/test_utils.py

```diff
@@ -108,7 +109,7 @@ def test_convert_from_pip(expected, requirement):
     ),
 ])
 def test_convert_from_pip_vcs_with_extra(expected, requirement):
-    assert pipenv.utils.convert_deps_from_pip(requirement) == expected
+    assert pipenv.requirements.PipenvRequirement.from_line(requirement).as_pipfile() == expected
```

Comment:
You can merge this feature into `DEP_PIP_PAIRS` to make sure the new parser can catch the extra part.

Reply:
nvm I’ll just push the change here.